packages/shared/src/polyfills/indexedDBShim/indexedDBShim.ts

```
@@ -1,45 +1,49 @@
+if (!global.IDBDatabase.prototype.transactionOriginal_a7c9d6a9) {
+  // @ts-ignore
+  global.IDBDatabase.prototype.transactionOriginal_a7c9d6a9 =
+    // eslint-disable-next-line @typescript-eslint/unbound-method
+    global.IDBDatabase.prototype.transaction;
+  global.IDBDatabase.prototype.transaction = function (
+    storeNames: string | string[],
+    mode?: IDBTransactionMode,
+    options?: IDBTransactionOptions,
+  ) {
+    // eslint-disable-next-line @typescript-eslint/no-this-alias
+    const self = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)